pytransform3d/rotations/_utils.py

```diff
@@ -59,8 +59,8 @@ def norm_angle(a):
     a_norm : float or array-like, shape (n,)
         Normalized angle(s) in radians
     """
-    # Source of the solution: http://stackoverflow.com/a/32266181
-    return -((np.pi - np.asarray(a)) % (2.0 * np.pi) - np.pi)
+    a = np.asarray(a, dtype=np.float64)
```

Comment:
Please use `dtype=float` here. This is what we use in other modules of the library.

Reply:
Also 2 * Pi should be a constant. We can define it  in the file _constants.py